exporter/containerimage/writer.go

```diff
@@ -369,6 +547,22 @@ func emptyImageConfig() ([]byte, error) {
 	return dt, errors.Wrap(err, "failed to create empty image config")
 }
 
+func emptyImageConfig() ([]byte, error) {
+	type image struct {
+		ocispecs.Image
+	}
+
+	img := image{
+		Image: ocispecs.Image{
+			Architecture: "none",
```

Comment:
I've seen some examples with "unknown/unknown" but don't know if it is actually a convention.

Reply:
I think `unknown/unknown` is more meanigful, I think I mentioned that in another comment also.

Reply:
I guess meaningfulness is just hard to improve, as it's just relative. I was thinking of "what platform this attestation applies to?", in which case "none" doesn't seem to be right as there is definetly a platform that it relates to, it just cannot be defined by this field directly. I would say actually "undefined" would be better then "unknown".
I get your point that this is not an image as such and it cannot run on any platform, but it still relates to an image and thereby bares its own notion of a platform...